docs/user/extension/mbstyle/index.rst

```diff
@@ -43,18 +43,19 @@ Zoom level tuning
 
 The mapping between zoom levels and scale denominators is performed assuming, by default, a 512 pixels
 root tile at zoom level zero. This matches the usage of vector tiles client side.
-If a different value is to be used, it's possible to set a system variable to change the default.
+If a different value is to be used, it's possible to set a system property to change the default.
 E.g., the following would match "x4" tiles:
 
 .. code-block:: java
 
-    System.setSystemProperty("MBSTYLE_ROOT_TILE_PIXELS", "1024") 
+   System.setSystemProperty("MBSTYLE_ROOT_TILE_PIXELS", "1024") 
 
 MapBox Types
 ^^^^^^^^^^^^
-  copied from the `MapBox Style Specification <https://www.mapbox.com/mapbox-gl-js/style-spec/>`_
 
-  Color
+Quoting `MapBox Style Specification <https://www.mapbox.com/mapbox-gl-js/style-spec/>`__ :
+
+  ``Color``
```

Comment:
@tbarsballe this section has some duplication with `spec/types.rst`, do you know any background to untangle?

Reply:
`spec/*.rst` is a direct copy of the MapBox spect with GeoTools supported version added.
This index page tries to provide a more concise overview / quickstart, and pulls some of the more top-level information from the spec. It could maybe do with some rewording, but should still keep all that info in it.

Reply:
okay I removed the duplication, and made this focus on the example snippets